src/dev/src/app/combobox/combobox-static.html

```diff
@@ -0,0 +1,76 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<form class="test-select">
+    <section class="form-block">
+        <label>Select Boxes</label>
+        <div class="form-group">
+            <label for="selects_1">This is a select box</label>
+            <div class="select">
+                <select id="selects_1">
```

Comment:
Why do we still have all these select demos for combobox? We should just delete them.